### ISSUE TITLE
zmin, zmax in delta_extraction not being used?

### ISSUE THREAD

**Issue by andreufont** [igmhub/picca#830]
The defaults are supposed to be zmin=2.1, zmax=3.5 (based on tutorial), but in the output file I read:
[PROGRESS]: and z >= 2.0        : nb object in cat = 265328
[PROGRESS]: and z < 4.288461538461538         : nb object in cat = 265328

It would be good to clarify what is going on.

**Comment by iprafols** [comment]
Hi @andreufont, 
The behavior on the master branch is to compute the `zmin` and `zmax` based on the limits of the specified wavelenght ranges. See lines 464-469 in `picca_deltas.py` for this:
```
if args.zqso_min is None:
    args.zqso_min = max(0., args.lambda_min / args.lambda_rest_max - 1.)
    userprint("zqso_min = {}".format(args.zqso_min))
if args.zqso_max is None:
    args.zqso_max = max(0., args.lambda_max / args.lambda_rest_min - 1.)
    userprint("zqso_max = {}".format(args.zqso_max))
```
This was a bit confusing when I first started the branch delta_extraction as the defaults in `read_data` from `io.py` are 2.1 and 3.5. So I started with these default values but later updated to computing them on the fly if not specified as it is currently done in the master branch. This now happens at the `__init__` method of `QuasarCatalogue` from `delta_extration/quasar_catalogue.py`. I simply forgot to update the tutorial.
